Automattic/jetpack
# Instant Search: Add focus styles.

#### Changes proposed in this Pull Request:


Add focus styles to the various overlay elements (excluding the search input field). Covers the Close box, the sort options, the "Load more" button, the "Clear filters" button, the checkboxes, and the hidden close button.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit a site with a Jetpack Search plan.
2. Open the Overlay by appending `?s=&category=uncategorized` to your site address.
3. Confirm the buttons look correct in both Light & Dark themes.
4. Toggle the light/dark themes from `/wp-admin/admin.php?page=jetpack-search-configure`.

https://user-images.githubusercontent.com/40267301/183593894-6616af60-c783-48be-976c-b4c96ccb28fe.mp4
